posydon/utils/datasets.py

```diff
@@ -0,0 +1,167 @@
+                          'description': "The collection of all POSYDON data "\
+                                         + "sets.",
+                          'md5': None,
+                          'title': "POSYDON community",
+                          'url': "https://zenodo.org/communities/posydon"
+                         }}
+COMPLETE_SETS = {}
+
+# auxiliary data
+ZENODO_COLLECTION['auxiliary'] = {
+    'data': None, #TODO
+    'description': "Auxiliary data for POSYDON. It contains data on "\
+                   + "supernova prescriptions (Sukhbold+2016, Couch+2020, "\
+                   + "Patton+Sukhbold2020), star formation history "\
+                   + "(IllustrisTNG), and detector sensitivity (of O3, O4low, "\
+                   + "O4high, design of three gravitational wave detectors "\
+                   + "H1, L1, V1).", #TODO
+    'md5': None, #TODO
+    'title': "Auxiliary POSYDON data", #TODO
+    'url': "https://zenodo.org/communities/posydon" #TODO
+}
+
+# v1
+ZENODO_COLLECTION['v1_for_v2.0.0-pre1'] = {
+    'data': "https://zenodo.org/record/14205146/files/POSYDON_data.tar.gz",
+    'description': "The POSYDON v1 dataset post-processed to work with the "\
+                   + "v2.0.0-pre1 version of the code. It contains "\
+                   + "single-HMS, single-HeMS, HMS-HMS, CO-HMS_RLO, CO-HeMS, "\
+                   + "CO-HeMS_RLO all at solar metallicity. Additionally, it "\
+                   + "includes data on supernova prescriptions "\
+                   + "(Sukhbold+2016, Couch+2020, Patton+Sukhbold2020), star "\
+                   + "formation history (IllustrisTNG), and detector "\
+                   + "sensitivity (of O3, O4low, O4high, design of three "\
+                   + "gravitational wave detectors H1, L1, V1).",
+    'md5': "cf645a45b9b92c2ad01e759eb1950beb",
+    'title': "Re-postprocessed POSYDON v1.0 dataset compatible with code "\
+             + "release v2.0.0-pre1",
+    'url': "https://zenodo.org/records/14205146"
+}
+ZENODO_COLLECTION['super-Eddington_v1'] = {
+    'data': "https://zenodo.org/records/14216817/files/"\
+            + "POSYDON_data_super_Eddington.tar.gz",
+    'description': "Data for super-Eddington accretion compatible with "\
+                   + "'v1_for_v2.0.0-pre1'. It contains CO-HMS_RLO, CO-HeMS, "\
+                   + "CO-HeMS_RLO all at solar metallicity to replace the "\
+                   + "corresponding data in 'v1_for_v2.0.0-pre1'",
+    'md5': "64b46fbd65cb54819371a1cbc714019c",
+    'title': "Re-postprocessed POSYDON v1.0 dataset, assuming "\
+             + "super-Eddington accretion, compatible with code release "\
+             + "v2.0.0-pre1",
+    'url': "https://zenodo.org/records/14216817"
+}
+COMPLETE_SETS['v1'] = ['v1_for_v2.0.0-pre1']
+COMPLETE_SETS['super-Eddington_v1'] = ['v1_for_v2.0.0-pre1',
+                                       'super-Eddington_v1']
+
+# v2
+ZENODO_COLLECTION['v2_girds_2Zsun'] = {
```

Comment:
```suggestion
ZENODO_COLLECTION['v2_grids_2Zsun'] = {
```

and likewise for other entries here.

Reply:
corrected.